energyplus_regressions/configure.py

```diff
@@ -0,0 +1,10 @@
+from plan_tools.entry_point import EntryPoint
+
+
+def configure_cli() -> None:
+    source_dir = "energyplus_regressions"
+    name = "energyplus_regression_runner"
+    description = "An EnergyPlus test suite utility"
+    nice_name = "EnergyPlus Regression Tool"
+    s = EntryPoint(source_dir, name, nice_name, description, name)
+    s.run()
```

Comment:
This file is a new executable entry point, which is used to create an entry_point shim during pip install.  Once you pip install this tool, you can run the new entry point configuration tool and it will add a desktop shortcut in a nice cross platform way.